cluster/addons/addon-manager/kube-addons-main.sh

```diff
@@ -62,7 +62,7 @@ done < <(find /etc/kubernetes/admission-controls \( -name \*.yaml -o -name \*.js
 log INFO "== Entering periodical apply loop at $(date -Is) =="
 while true; do
   start_sec=$(date +"%s")
-  if is_leader || is_cloud_leader; then
```

Comment:
We shouldn't be doing this. KAM piggy backs on KCM lease. If we use *OR* statement here, in case of multiple masters we can run two KAM on two instances: one where KCM is leader, the other where CCM is leader.